### PR TITLE
Logged transfer speed

### PR DESCRIPTION
**Pull request by Hyrumthehotscott**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Transfer.java [hunk 1]
```diff
@@ -85,8 +86,19 @@ public void setTransferNeutralOutput() {
     transferMotor.setControl(new NeutralOut());
   }
 
+  private double getTransferMotorVelocity() {
+    return transferMotor.getVelocity().getValueAsDouble();
+  }
+
+  private double getFeederMotorVelocity() {
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Transfer.java]: This is technically not velocity. It's a percentage of maximum output.

File: src/main/java/frc/robot/subsystems/Transfer.java [hunk 2]
```diff
@@ -85,8 +86,19 @@ public void setTransferNeutralOutput() {
     transferMotor.setControl(new NeutralOut());
   }
 
+  private double getTransferMotorVelocity() {
+    return transferMotor.getVelocity().getValueAsDouble();
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Transfer.java]: Since we are not commanding velocity, we should request percent output

**Review by TaylerUva — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Transfer.java [hunk 3]
```diff
@@ -85,8 +86,19 @@ public void setTransferNeutralOutput() {
...
+
+  private double getFeederMotorPercentOutput() {
+    return feederMotor.getMotorOutputPercent();
+  }
+
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    SmartDashboard.putNumber("Transfer/Left/Velocity RPS", getFeederMotorPercentOutput());
+    SmartDashboard.putNumber("Transfer/Right/Velocity RPS", getTransferMotorPercentOutput());
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Transfer.java]: Do these keys make sense?

**Review by TaylerUva — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Transfer.java [hunk 4]
```diff
@@ -85,8 +86,19 @@ public void setTransferNeutralOutput() {
...
+
+  private double getFeederMotorPercentOutput() {
+    return feederMotor.getMotorOutputPercent();
+  }
+
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    SmartDashboard.putNumber("Transfer/Feeder/Percent", getFeederMotorPercentOutput());
+    SmartDashboard.putNumber("Transfer/transfer/Percent", getTransferMotorPercentOutput());
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Transfer.java]:
```suggestion
    SmartDashboard.putNumber("Transfer/Percent", getTransferMotorPercentOutput());
```

File: src/main/java/frc/robot/subsystems/Transfer.java [hunk 5]
```diff
@@ -85,8 +86,19 @@ public void setTransferNeutralOutput() {
...
+
+  private double getFeederMotorPercentOutput() {
+    return feederMotor.getMotorOutputPercent();
+  }
+
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    SmartDashboard.putNumber("Transfer/Feeder/Percent", getFeederMotorPercentOutput());
+    SmartDashboard.putNumber("Transfer/Transfer/Percent", getTransferMotorPercentOutput());
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Transfer.java]:
```suggestion
    SmartDashboard.putNumber("Transfer/Percent", getTransferMotorPercentOutput());
```

**Review by TaylerUva — APPROVED** [review]

**Review by TaylerUva — APPROVED** [review]

**Review by alicekuznetsov — APPROVED** [review]
If T-dawg thought it was good it has to be good

**Pull request merged**